Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1023,16 +1025,16 @@ public static void HandleCI(Session session, params string[] parameters)
                 return;
             }
             var loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
```

Comment:
The loot generation needs to be move up into the Try, to prevent invalid weenies from crashing the server.